src/main/java/com/bravo/user/service/UserService.java

```diff
@@ -39,13 +41,37 @@ public UserReadDto create(final UserSaveDto request){
     return resourceMapper.convertUser(user);
   }
 
-  public UserReadDto retrieve(final String id){
+  public UserReadDto retrieveById(final String id){
     Optional<User> optional = userRepository.findById(id);
     User user = getUser(id, optional);
 
     LOGGER.info("found user '{}'", id);
     return resourceMapper.convertUser(user);
   }
+  
+  /**
+   * Retrieves a paginated list of users whose combined first, middle, and last names matches the specified name.
+   * 
+   * @param name Name to find with format [firstName][middleName][lastName] and no space. May contain negation ("!") or
+   * wildcard ("*"/"%") characters.
+   * @param pageRequest {@linkplain PageRequest} containing pagination parameters
+   * @param response {@linkplain HttpServletResponse} in which to store page information
+   * @return List of {@linkplain UserReadDto}s whose combined names match the search name criteria
+   */
+  public List<UserReadDto> retrieveByName(
```

Comment:
This method is unnecessary. The code almost completely duplicates `retrieve`. If the `UserFilter` was created in the `UserController` then the same method could've been used.

Reply:
Yep, will update.